migrations/363fc0301235_diff_phid_added.py

```diff
@@ -0,0 +1,30 @@
+"""Diff PHID added
```

Comment:
I think I will simply remove the phid step, but if someone had it already in sqlite than it would raise an error...